_drafts/Introducing-geoextent.md

```diff
@@ -0,0 +1,131 @@
+# Introducing geoextent
```

Comment:
Please add required metadata into the header, i.e. the title, authors, keywords. See files in `_posts` for examples.

Reply:
Done.